src/actionlist/action-list-item.scss

```diff
@@ -306,16 +305,21 @@
   // sizes
 
   &.ActionList-content--sizeMedium {
-    grid-template-rows: minmax($actionList-item-height-md, max-content); // 28px with 12px padding = [40px total]
+    // 44px total height
+    // stylelint-disable-next-line primer/spacing
+    padding: $actionList-item-padding-vertical-md $actionList-item-padding-horizontal;
```

Comment:
@vdepizzol per our review on spacing, here's another solution. I don't think its as elegant as the row height definitions but I think this makes more sense for the use case!
- grid row will always be minimum `min-content`
- padding top/bottom changes per size variant

I think I'm seeing the typography struggles you were having. I had to set an explicit line-height value on the `label` and I'm not sure that's a good solution. If I don't do that, the value will be 1 pixel off (or several) and the math is funky.

### small (default)
| no desc | desc |
| :- | :- |
| ![image](https://user-images.githubusercontent.com/18661030/139508987-93ea0738-feb9-4f09-b0cf-ea7d5d82c7f5.png) | ![image](https://user-images.githubusercontent.com/18661030/139509007-1fbd405b-92b9-417c-8dcb-3a5c05896885.png) |

### medium
| no desc | desc |
| :- | :- |
| ![image](https://user-images.githubusercontent.com/18661030/139509059-a620f45d-4cb1-4609-9245-f98a86023289.png) | ![image](https://user-images.githubusercontent.com/18661030/139509122-292f4746-a743-48fd-a767-dbf7f8f97ff9.png) |

### large
| no desc | desc |
| :- | :- |
| ![image](https://user-images.githubusercontent.com/18661030/139508870-e1f4c851-7cd0-40a7-aafa-addb8a287f30.png) | ![image](https://user-images.githubusercontent.com/18661030/139508841-8574f913-64e0-494e-8a65-d902386a76a2.png) |

These values should match what I'm seeing in figma 🤞

Reply:
@langermank I think this makes sense! Thanks for revisiting this implementation! 🙇